api/Navigator.json

```diff
@@ -3784,12 +3796,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "100",
-                "notes": [
-                  "Version 107 and later, API use is gated by a site permission add-on that is automatically generated by Firefox.",
-                  "Version 105 and later, the <code>dom.webmidi.gated</code> preference may be set <code>false</code> to allow WebMIDI to be used without a site permission addon.",
-                  "Version 100 to 104 (inclusive) WebMIDI is not available (API use is gated by a site permission add-on that is no longer available)."
-                ]
+                "version_added": "100"
```

Comment:
```suggestion
                "version_added": "108"
```

Reply:
@queengooborg I guess this is because the value isn't flagged. Now I'm going to be inconsistent and merge this, because adding a flag here doesn't seem worth it.